### PR TITLE
Add `newaxis` to torch aliases

### PR DESCRIPTION
**Pull request by rgommers**
Closes gh-64

### CODE REVIEW

**Comment by rgommers** [comment]
Will add a corresponding test case to `array-api_tests` as well.

**Comment by asmeurer** [comment]
Looks good. Merging might be blocked on #65 though.

File: array_api_compat/torch/_aliases.py [hunk 1]
```diff
@@ -694,7 +695,7 @@ def take(x: array, indices: array, /, *, axis: Optional[int] = None, **kwargs) -
            'atan2', 'bitwise_and', 'bitwise_left_shift', 'bitwise_or',
            'bitwise_right_shift', 'bitwise_xor', 'divide', 'equal',
            'floor_divide', 'greater', 'greater_equal', 'less', 'less_equal',
-           'logaddexp', 'multiply', 'not_equal', 'pow', 'remainder',
+           'logaddexp', 'multiply', 'newaxis', 'not_equal', 'pow', 'remainder',
```
Comment (asmeurer) [inline array_api_compat/torch/_aliases.py:698]: I've tried to keep these in the same order they are in the file (although I see now that that isn't really completely true).

**Comment by rgommers** [comment]
This was green before, and gh-65 is unrelated and should not be blocking. No idea why the `fft` tests are failing now, probably another case of "should use derandomize". Either way, let's merge this?

**Comment by asmeurer** [comment]
We'll have to fix the CI before we release at least.

**Pull request closed without merging**